Functions/New-MockObject.ps1

```diff
@@ -0,0 +1,51 @@
+function New-MockObject {
+    <#
+    .SYNOPSIS
+        This function instantiates a .NET object from a type. The assembly for the particular type must be
+        loaded.
+
+    .PARAMETER Type
+        The .NET type to create an object from.
+
+    .EXAMPLE
+        New-MockObject -Type 'System.Runtime.Serialization.Formatterservices'
```

Comment:
The example fails with: Cannot create abstract class. The next example on line 13 and 14 is imho unnecessary. Wrapping the call to GetUninitializedObject is the goal of your function, so why expose it's internals. I suggest you make the example `System.Diagnostics.Process`. And show it's output, if you have to: 
```powershell
New-MockObject -Type 'System.Diagnostics.Process'

Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id  SI ProcessName                                                                                                                                                                            
-------  ------    -----      ----- -----   ------     --  -- -----------                                                                                                                                                                            
              0        0          0     0                                    
```